decidim/decidim
# Fix admin redesign in templates module

#### :tophat: What? Why?
This PR solves some of the design issues signaled in #11606

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11606

#### Testing

1. Login as admin and visit templates admin panel 
2. Check the elements in the associated ticket 
3. Apply the patch 
4. See the elements in the list being changed 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
